btcjson/chainsvrwsresults.go

```diff
@@ -8,3 +9,21 @@ package btcjson
 type SessionResult struct {
 	SessionID uint64 `json:"sessionid"`
 }
+
+// RescanBlocksResult models the result object returned by the rescanblocks RPC.
+//
+// NOTE: This is a btcsuite extension ported from
+// github.com/decred/dcrd/dcrjson.
+type RescanBlocksResult struct {
+	DiscoveredData []RescannedBlock `json:"discovereddata"`
```

Comment:
ACK. I've changed this PR to reflect that, and also changed the matching `btcrpcclient` PR.